File: jar/src/main/java/jline/api/sn/SnHasSJF.java
package jline.api.sn;

import jline.lang.NetworkStruct;
import jline.lang.constant.SchedStrategy;

public final class SnHasSJF {
    private SnHasSJF() {}

    public static boolean snHasSJF(NetworkStruct sn) {
        return sn.sched.containsValue(SchedStrategy.SJF);
    }
}

File: jar/src/main/kotlin/jline/api/sn/SnHasSJF.kt
package jline.api.sn

import jline.lang.NetworkStruct
import jline.lang.constant.SchedStrategy

fun snHasSJF(sn: NetworkStruct): Boolean {
    return sn.sched.containsValue(SchedStrategy.SJF)
}
/**
 * Stochastic network HasSJF algorithms
 */
@Suppress("unused")
class SnhassjfAlgo {
    companion object {
        // Class documentation marker for Dokka
    }
}